bench/bench.ml

```diff
@@ -76,9 +76,8 @@ module Package = struct
     let stdout_to = Process.Io.make_stdout Swallow in
     let stderr_to = Process.Io.make_stderr Swallow in
     let stdin_from = Process.Io.(null In) in
-    Process.run Strict
-      ~display:!Dune_engine.Clflags.display
-      ~stdout_to ~stderr_to ~stdin_from (Lazy.force git)
+    Process.run Strict ~display:Quiet ~stdout_to ~stderr_to ~stdin_from
+      (Lazy.force git)
       [ "clone"; uri t ]
```

Comment:
Here we have the bench script running git to clone the dune universe repo. We don't need to see this reported to the user.